src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -252,11 +261,10 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
             Choose how you&apos; d like your {ROOMS_TAXON.lower} to appear on
             the map
           </div>
-          <Form.Control as="select" custom name="roomVisibility" ref={register}>
-            <option value="hover">Hover</option>
-            <option value="count">Count</option>
-            <option value="count/name">Count and names</option>
-          </Form.Control>
+          <PortalVisibility
+            updateRoomVisibility={updateRoomVisibility}
+            visibilityState={venue.roomVisibility}
+          />
```

Comment:
Same comment like the previous uses of this component.

Reply:
ref https://github.com/sparkletown/sparkle/pull/2449#discussion_r742024970